analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/DisablingRequestValidationTest.cs

```diff
@@ -39,33 +39,41 @@ public class DisablingRequestValidationTest
     {
         private const string AspNetMvcVersion = "5.2.7";
         private const string WebConfig = "Web.config";
+        private const string SonarProjectConfig = "SonarProjectConfig.xml";
```

Comment:
This can be removed after addressing the other comment, as it will be needed only on a single place.